test/fixtures/e2e/amp-form.ssr.html

```diff
@@ -0,0 +1,315 @@
+<!doctype html>
```

Comment:
I think we should put tests in a per-component folder in the `test/fixtures/e2e` if they're for a specific component, e.g. `test/fixtures/e2e/amp-form`. I can see there being other e2e tests for `amp-form`. What do you think?

Reply:
sgtm. I thought about making that change but decided not to until that time came. But I plan on following up with a PR to migrate all existing e2e tests here, so I'll do it now rather than later. Thanks